front/app/modules/commercial/customizable_homepage_banner/admin/CTASettings/CustomizedButtonSettings.tsx

```diff
@@ -12,6 +12,16 @@ import { injectIntl, FormattedMessage } from 'utils/cl-intl';
 import { InjectedIntlProps } from 'react-intl';
 import useAppConfigurationLocales from 'hooks/useAppConfigurationLocales';
 import { isNilOrError } from 'utils/helperUtils';
+import styled from 'styled-components';
+
+const Container = styled(SectionField)`
+  margin-left: 28px;
+`;
+
+const TextSettings = styled.div`
+  margin-top: 10px;
+  margin-bottom: 15px;
```

Comment:
Another recommendation to save later trouble is to add margin only in 1 direction. We normally do margin right and margin bottom. Not margin bottom **and** margin top. It might clash with components above their margin bottom.

Reply:
In this case, unfortunately, margin bottom doesn't seem to be convenient because I would need to add it to the label in SettingRadioButtons.tsx. And to add a new ugly condition for it.

Reply:
And again, this component seems unlikely to be reused.